projects/adapter/src/dbt/adapters/fal_experimental/utils/environments.py

```diff
@@ -137,81 +133,30 @@ def create_environment(
     config: Dict[str, Any],
     machine_type: str = "S",
     credentials: Optional[Any] = None,
-):
-    from isolate.backends.virtualenv import VirtualPythonEnvironment
-    from isolate.backends.conda import CondaEnvironment
-    from isolate.backends.remote import IsolateServer
-
-    parsed_config = {
-        key: val for key, val in config.items() if key not in CONFIG_KEYS_TO_IGNORE
-    }
-
-    if credentials.key_secret and credentials.key_id:
-        return _build_hosted_env(config, parsed_config, credentials, machine_type)
-
-    REGISTERED_ENVIRONMENTS: Dict[str, BaseEnvironment] = {
-        "conda": CondaEnvironment,
-        "venv": VirtualPythonEnvironment,
-        "remote": IsolateServer,
-    }
-
-    env_type = REGISTERED_ENVIRONMENTS.get(kind)
-
-    if env_type is None:
+) -> EnvironmentDefinition:
+    if kind not in ["venv", "conda"]:
```

Comment:
should we support `virtualenv` too? Since that is the remote version?

Reply:
`venv` is `virtualenv`, for some reason we rename in `isolate`. Should come to a decision about sticking to one and ditching the other.